en/concept/events_interface.md

```diff
@@ -0,0 +1,133 @@
+# Events Interface
+
+The *Events Interface* provides a system-wide API to inform a user about certain events.
```

Comment:
Perhaps a little more background and preparation for the existence of the MAVLink docs. For example, some examples would be nice. Maybe something like the below.

---

The *Events Interface* provides a system-wide API for notification of events, which are published to GCSs via the MAVLink Events Service [LINK WHEN AVAILABLE] (to GCSs) and also stored in system logs [LINK TO LOGS DOCS].

The interface can be used for publishing events for state changes or any other type of occurrence, including things like arming readiness, [WHAT ELSE DO WE USE STATUS TEXT FOR?]

:::note
The events interface will replace the use of `mavlink_log_*` calls in PX4 code, (and `STATUS_TEXT` messages in MAVLink) for event notification following (???) PX4 v1.12.
There will be an intermediate period where both approaches are supported [LINK TO SECTION BELOW ON BACKWARDS COMPAT]
:::

Reply:
@bkueng Does the MAVLink spec cover the concept of common/managed enums? Specifically, if it is likely that a GCS will configure itself based on events (say display "ready to ARM" when a readiness event is received) then it makes sense that this kind of event is shared between GCS so that nothing needs to be coded twice. 

@julianoes Perhaps madness, but could/should we reasonably use events as a generic mavlink notifier "in future"? What I mean is that for command protocol we're tdiscussing mechanisms to tell us completion of long running events. We could instead bypass completely and define a Takeoff complete event. I appreciate this depends on events interface being broadly implemented, but is the idea "sane"?

Reply:
Yes, certainly. Ideally we'll link to the (not-yet-existing) mavlink docs

Reply:
OK, I have updated this section, please check.